src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int bound = 100;
+    private final Random random = new Random();
+    private final int randomNumber = random.nextInt(bound);
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private String color = String.valueOf(colorSupplier.getRandomColor());
```

Comment:
This is not correct. Now your number and color is initialized when application starts and never changes. But you should generate them every time method 'getRandomBall' is invoked
```suggestion
    private final Random random = new Random();
    private ColorSupplier colorSupplier = new ColorSupplier();
```